Code changes done for opening Typdef and Class private data in run time

shivamanikandan opened the pull request:
Code changes are done for Opening TypeDef. and Class private data from block diagram (constants, controls & indicators) in run time.

Please review and LMK if you find bugs or improvements

ItsMeAshiq commented on src/code/open-typedef-class.llb/Test Main.vi:
Remove dependency on LUnit Class Object & replace it with a test class object.

ItsMeAshiq commented on src/code/open-typedef-class.llb/Testv2.vi:
Redundant file.

shivamanikandan commented:
Removed redundant files
Moved VIs, Ctls to src\tests which are used only for testing
Created custom class to avoid the dependency with VI packages
Added all the files related open-typedef-class.llb to Project.lvproj itself
